source/Calamari.Tests/Fixtures/PackageRetention/Repository/JournalInMemoryRepository.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using Calamari.Common.Plumbing.Deployment.PackageRetention;
+using Calamari.Deployment.PackageRetention.Model;
+using Calamari.Deployment.PackageRetention.Repositories;
+
+namespace Calamari.Tests.Fixtures.PackageRetention.Repository
+{
+    public class JournalInMemoryRepository : IJournalRepository
```

Comment:
Reversed compared to what I'm used to `InMemoryJournalRepository`

Reply:
Definitely reads better your way.